presto-docs/src/main/sphinx/release/release-0.259.1.rst

```diff
@@ -0,0 +1,3 @@
+===============
+Release 0.259.1
+===============
```

Comment:
JDBC Connector Changes
_______________
* Revert partial pushdown of JDBC filters (:pr:`16412`)